### PR TITLE
Completed css-two-column-layout exercise-Dulce Herrera

### PR DESCRIPTION
**Pull request by dulceherrera**
Screen Shot 2022-07-02 at 11 54 27 AM

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-half {
+  flex-basis: 50%;
+}
+
+.column-one-fourth {
+  flex-basis: 30%;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:30]: This should be `25%`

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  color: #1c1c1c;
+  width: 250px;
+}
+
+.row .column-full {
+  justify-content: center;
+  width: 800px;
+  align-content: center;
+  flex-wrap: wrap;
+}
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:67]: If you feel like you need to make a `column` class into a flexbox, that usually means you just need a child `row` inside that `column`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-two-column-layout/styles.css (hunk 1 above)
Comment (dulceherrera) [reply css-two-column-layout/styles.css]: Screen Shot 2022-07-09 at 2 51 11 PM

File: css-two-column-layout/styles.css (hunk 2 above)
Comment (dulceherrera) [reply css-two-column-layout/styles.css]: Screen Shot 2022-07-09 at 2 50 46 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**